specs/Modal.spec.js

```diff
@@ -166,6 +166,7 @@ describe('State', () => {
     expect(
       moverlay(modal).className.includes('myOverlayClass')
     ).toBeTruthy();
+    unmountModal();
```

Comment:
Yeah I'll remove, it was failing during one of my initial attempts, but it's passing now with the latest updates 👍